fission-core/library/Fission/Error/GenericError/Types.hs

```diff
@@ -4,4 +4,5 @@ module Fission.Error.GenericError.Types (GenericError (..)) where
 import           Fission.Prelude
 
 newtype GenericError = GenericError String
-  deriving (Eq, Show, Exception)
+  deriving stock    (Eq, Show)
+  deriving anyclass (Exception)
```

Comment:
This is just a compiler warning. It said that it was inferring stock & anyclass here, so I just made it explicit.